app/src/main/java/com/dimelo/sampleapp/RcConfigurationActivity.java

```diff
@@ -0,0 +1,47 @@
+package com.dimelo.sampleapp;
+
+import android.os.Bundle;
+import android.support.design.widget.TextInputLayout;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.Switch;
+
+import com.dimelo.dimelosdk.main.Dimelo;
+
+public class RcConfigurationActivity extends AppCompatActivity {
+    Dimelo dimelo;
+    Switch switchCompat;
```

Comment:
```suggestion
    private Switch switchCompat;
```

Reply:
why private and by default is protected?

Reply:
Because this var will be used only in this class so no need to be protected (protected will be visible in others class in the same package name)